examples/mnist_gans/runner.py

```diff
@@ -0,0 +1,68 @@
+from typing import Mapping, Any
+
+import torch
+from catalyst.dl import Runner
+
+
+class GANRunner(Runner):
```

Comment:
maybe it's better to move `GANRunner` from examples into `catalyst/dl/runner` ?

Reply:
OK

Actually right now to implement some other GAN basing on current vanilla GANRunner someone will have to override it almost completely. I suppose we will have to think about better/more generic/easy extendable implementation which will be more useful parent class.

But I'm going to left this important task to other PR's =)